src/libstd/net/addr.rs

```diff
@@ -457,6 +459,15 @@ impl ToSocketAddrs for str {
     }
 }
 
+#[stable(feature = "slice-to-socket-addrs", since = "1.8.0")]
```

Comment:
Currently we require feature names to be valid rust identifiers, so could this change `-` to `_`?